internal/globaldefault/default.go

```diff
@@ -61,6 +98,31 @@ func Prepare(subshell subshell.SubShell) error {
 	return nil
 }
 
+// WarningForAdministrator prints a warning message if default activation is invoked by a Windows Administrator
+// The default activation will only be accessible by the underlying unprivileged user.
+func WarningForAdministrator(out output.Outputer) {
+	if rt.GOOS != "windows" {
+		return
+	}
+
+	isAdmin, err := osutils.IsWindowsAdmin()
+	if err != nil {
+		logging.Error("Failed to determine if run as administrator.")
+	}
+	if isAdmin {
+		u, err := user.Current()
+		if err != nil {
+			logging.Error("Failed to determine current user.")
+			return
+		}
+		out.Notice(locale.Tl(
+			"default_admin_activation_warning",
+			"[NOTICE]The default activation is added to the environment of user {{.V0}}.  The project may be inaccessible when run with Administrator privileges or authenticated as a different user.[/RESET]",
+			u.Username,
+		))
+	}
+}
```

Comment:
While this needs issue definitely needs to be discussed and addressed I feel this code is well outside the scope of this story. We are introducing new messaging to the user that should be vetted by both Nathan and Pete before it goes in. It may be best to add this to the ticket description [here](https://www.pivotaltracker.com/n/projects/2203557/stories/175826240) as a possible solution or stop-gap until we know how we want to handle it.

If you like to get this in on this PR we can involve Pete and see what he thinks.

Reply:
Good point.  I'll remove it, and bring it up during stand-up anyways.